### ISSUE TITLE
Please tag the releases

### ISSUE THREAD

**Issue by PatrikN-work** [eclipse-theia/theia-blueprint#303]
### Feature Description:
Please tag the releases. The last tag is 3 years old.

**Comment by marcdumais-work** [comment]
Indeed, we have an initial `v0.0.1` tag in the repo, and that's it:
![image](https://github.com/eclipse-theia/theia-blueprint/assets/25749063/0647d5bc-aae4-498f-a193-50fde8234985)

Related - ATM we only manually step the [version of the Blueprint app](https://github.com/eclipse-theia/theia-blueprint/blob/master/applications/browser/package.json#L6), when we update the Theia version. Between these updates, any PR merged, we release new Blueprint packages, that re-uses the same version number. Probably we should step that version, so it's possible to distinguish from exactly which commit/tag, a given Blueprint package was built. Also it probably would not hurt to include the version in the package names.

To recap:
- step Blueprint version in each app's `package.json` for each release. e.g. use Theia's version as is done now, but step the minor version for each merged PR
- add a tag in the repo for each release
- include version in package names

**Comment by marcdumais-work** [comment]
cc: @tsmaeder @planger WDYT?

**Comment by marcdumais-work** [comment]
As a temporary work-around, are there any objections, to me manually adding a _git tag_ for 1.41.0?

**Comment by planger** [comment]
I'm not really familiar with the current release process of Blueprint, probably @jfaltermeier should chime in on that!
In general I don't have any objection to tag manually for now. But on the long run, it'd be good to have an automated workflow that creates a Github release and that tags the git ref from which the version has been built.

**Comment by jfaltermeier** [comment]
Currently each commit on master triggers a build that will be released. 
I agree that for now we can add tags manually, but we should look for an automated solution including version increments for each published version.

**Comment by marcdumais-work** [comment]
release tag manually added: [v1.41.0](https://github.com/eclipse-theia/theia-blueprint/releases/tag/v1.41.0)
